src/com/gh4a/activities/SearchActivity.java

```diff
@@ -155,6 +186,24 @@ public void onCreate(Bundle savedInstanceState) {
         mSearch.setOnCloseListener(this);
         mSearch.onActionViewExpanded();
 
+        mSearch.setOnSuggestionListener(this);
+        mSearch.setSuggestionsAdapter(
+                new SimpleCursorAdapter(this, android.R.layout.simple_list_item_1, null,
+                        new String[]{SuggestionsProvider.Columns.SUGGESTION},
+                        new int[]{android.R.id.text1}, 0)
+        );
+
+        final SearchView.SearchAutoComplete complete = (SearchView.SearchAutoComplete) mSearch.findViewById(R.id.search_src_text);
+        complete.setThreshold(0);
```

Comment:
I'm not sure whether I'm a fan of this; autocompleting starting before the first character feels somewhat overzealous to me.  IMHO the default of 2 is better.